docs/sql-performance-tuning.md

```diff
@@ -342,4 +371,33 @@ Data skew can severely downgrade the performance of join queries. This feature d
        </td>
        <td>3.0.0</td>
      </tr>
+     <tr>
+       <td><code>spark.sql.adaptive.forceOptimizeSkewedJoin</code></td>
```

Comment:
This is a non-internal configuration which has been defined and used for a while.

https://github.com/apache/spark/blob/786594734bd79017ebd42eb117b62958afad07bb/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L801-L806